python/ray/tune/integration/kubernetes.py

```diff
@@ -0,0 +1,143 @@
+import kubernetes
+import subprocess
+
+from ray import services, logger
+from ray.autoscaler.command_runner import KubernetesCommandRunner
+from ray.tune.syncer import NodeSyncer
+from ray.tune.sync_client import SyncClient
+
+
+def NamespacedKubernetesSyncer(namespace):
+    """
+    Wrapper to return a ``KubernetesSyncer`` for a specific
+    Kubernetes namespace.
```

Comment:
btw, first line should be on same line as quotes - it's the "summary line".

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html